visualization/app/codeCharta/ui/nodeContextMenu/nodeContextMenu.component.ts

```diff
@@ -88,6 +90,10 @@ export class NodeContextMenuController
 		this.onShowNodeContextMenu(building.node.path, nodeType, x, y)
 	}
 
+	public onMapRightClick(x: number, y: number) {
+		this.hideNodeContextMenu()
+	}
```

Comment:
add a tests, that checks if `this._viewModel.codeMapNode` is null afterwards

Reply:
The event passes click coordinates (x and y) that are not used in this particular Event Handler. The Linter will break the build.

Reply:
Yes. Simply remove them.